packages/neuron-wallet/tests/test-utils.ts

```diff
@@ -1,4 +1,25 @@
+import { EventEmitter } from 'events'
+
 export const flushPromises = () => {
   jest.runAllImmediates()
   return new Promise(setImmediate)
 };
+
+
+export class SyncTask extends EventEmitter {
```

Comment:
Maybe we can use `jest.doMock`, so we don't need to manually create this mock class?

Reply:
Actually, we don't need this mock anymore, as the main process won't import `SyncTask` once the entire sync task has been moved to a child thread.